docs/how-to/hip_porting_guide.rst

```diff
@@ -120,22 +180,23 @@ CONVERTED refs by names:``). It also lists the total lines of code for the file
 and potential warnings. In the end it prints a summary for all files.
 
 Automatically converting a CUDA project
---------------------------------------------------------------------------------
+---------------------------------------
 
 To directly replace the files, the ``--inplace`` option of ``hipify-perl`` or
 ``hipify-clang`` can be used. This creates a backup of the original files in a
 ``<filename>.prehip`` file and overwrites the existing files, keeping their file
 endings. If the ``--inplace`` option is not given, the scripts print the
 hipified code to ``stdout``.
 
-``hipconvertinplace.sh``or  ``hipconvertinplace-perl.sh`` operate on whole
+``hipconvertinplace.sh`` or ``hipconvertinplace-perl.sh`` operate on whole
 directories.
 
-Library Equivalents
---------------------------------------------------------------------------------
+Library equivalents
+===================
 
 ROCm provides libraries to ease porting of code relying on CUDA libraries.
-Most CUDA libraries have a corresponding HIP library.
+Most CUDA libraries have a corresponding HIP library. For more information,
+see either :doc:`ROCm libraries <rocm:reference/api-libraries>` or :doc:`HIPIFY CUDA compatible libraries <hipify:reference/supported_apis>`.
 
 There are two flavours of libraries provided by ROCm, ones prefixed with ``hip``
 and ones prefixed with ``roc``. While both are written using HIP, in general
```

Comment:
> ones prefixed with ``hip`` and ones prefixed with ``roc``

Is the "ones ... and ones ..." expression used in English? I've never heard that one before, but I'm not a native speaker.

Reply:
It is not uncommon, but I have replaced it with libraries.